protocol/oracle-manager/README.md

```diff
@@ -42,6 +42,8 @@ The Uniswap Node retrieves data from a [Uniswap Oracle](https://docs.uniswap.org
 - Parameters:
   - `address tokenAddress` - The address of the token
   - `address stablecoinAddress` - The address of the stablecoin
+  - `uint8 tokenDecimals` - The number of decimals places used by the token contract. _This must match what is provided by the token contract's `decimals()` function_.
+  - `uint8 stablecoinDecimals` - The number of decimals places used by the stablecoin contract. _This must match what is provided by the stablecoin contract's `decimals()` function_.
```

Comment:
is it always the case that this node has to use a `stablecoin` address? What if you want to get a uniswap price between two non-stablecoins?

in uniswap, they use a different strategy: they call it `token0` and `token1`. Perhaps we should follow the same pattern here?

Reply:
But we need to resolve it to a denominated value with 18 decimal precision?